spec.bs

```diff
@@ -1819,6 +1831,49 @@ Several APIs specific to fenced frames are defined on the {{Fence}} interface.
   </wpt>
 </div>
 
+<div algorithm>
+  The <dfn method for=Fence>notifyEvent(|event|)</dfn> method steps are:
+
+  1. If [=this=]'s {{Document}} is not [=Document/fully active=], then return.
+
+  1. Let |navigable| be [=this=]'s [=relevant global object=]'s [=Window/navigable=].
+  
+  1. If any of the following conditions are met, throw a {{SecurityError}} {{DOMException}}:
+
+      * |navigable| is not a [=fenced navigable container/fenced navigable=];
+
+      * |event| is null;
```

Comment:
I don't think `event` is nullable in the current PR (where `event` is optional), and it definitely is not if we adopt the changes I suggest above about making the parameter optional.

Reply:
I deleted this line, since I made the change you specified above about no optional/default value being provided.